src/components/MAPI/releases/ClusterDetailReleaseDetailsModal.tsx

```diff
@@ -71,9 +77,24 @@ const ClusterDetailReleaseDetailsModal: React.FC<IClusterDetailReleaseDetailsMod
 
         {releaseNotesURL && (
           <StyledReleaseDetailsModalSection title='Release notes'>
-            <a href={releaseNotesURL} rel='noopener noreferrer' target='_blank'>
-              {releaseNotesURL}
-            </a>
+            <Text>
```

Comment:
Leftover from the previous "release details modal" PR. This had the wrong font size.